src/time.rs

```diff
@@ -1,11 +1,21 @@
-use chrono::{Local, NaiveDate};
+use chrono::{Local, NaiveDate, NaiveTime};
 
 use crate::{Date, DateTime};
 
 pub fn now() -> DateTime {
     Local::now().naive_local()
 }
 
+pub fn one_sec_before_midnight() -> DateTime {
+    let midnight = NaiveTime::from_hms_opt(23, 59, 59);
+    Local::now().date_naive().and_time(midnight.unwrap())
```

Comment:
Avoid using unwrap here, use `?`.

Reply:
Got it.  Already resubmit PR with Option using ? instead of unwrap.